pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -405,41 +422,6 @@ func publishVpnConnection(vpnInfo *zinfo.ZInfoVpn,
 	return vpnConnInfo
 }
 
-func publishInfo(ctx *zedagentContext, UUID string, infoMsg *zinfo.ZInfoMsg) {
-	publishInfoToZedCloud(UUID, infoMsg, ctx.iteration)
-	ctx.iteration += 1
```

Comment:
Where do we update and use an iteration counter with the new approach? This is used to round-robin across multiple management ports thus with eth0 and eth1 both management ports we should see about the same number on calls to the /info API on each. You can check that in the zedcloud part of the zcli edge-node show --raw metrics, or look inside those metrics on EVE.